### PR TITLE
Update README.md - add groupedbar example

### PR DESCRIPTION
**Pull request by kaslusimoes**
Just adding the example @mkborregaard built from the discussion on [discourse](https://discourse.julialang.org/t/plots-bars-side-by-side/6655/4)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -204,6 +204,20 @@ The `group` syntax is also possible in combination with `groupedbar`:
 ```julia
 groupedbar([1, 2, 1, 2, 1, 2], rand(6), group = [1, 1, 2, 2, 3, 3])
 ```
+A more complex example is also possible:
```
Comment (mkborregaard) [inline README.md]: Can you delete this and the example before? And consider whether one of the two previous graphs could be taken out as well? I don't think we need more than two plots for groupedbar, and they should ideally be combined to a two-panel plot (perhaps reusing the data, but letting e.g. one represent the column-matrix attributes and the minimum input, the other the grouped method).

**Comment by mkborregaard** [comment]
Thanks. It'd be great, once we're editing it, to rethink all of the groupedbar example rather than just stacking new info on top.

File: README.md (hunk 1 above)
Comment (kaslusimoes) [reply README.md]: Sorry, I cannot understand the example you are giving. Can you please elaborate further?

Either way I already deleted this line and the example before.

**Comment by mkborregaard** [comment]
Totally forgot about this, sorry. Thanks!

**Comment by kaslusimoes** [comment]
Hehe that's no problem. Thank you for merging!

**Pull request merged**